### DISCUSSION TITLE
ORD INDEX - Upload Safe?

### DISCUSSION THREAD

**Discussion by DEFSTALKR** [ordinals/ord#1522] [Ideas]
After my horrid experience with **ord index** (took me a week with endless errors/dropouts lol), I'm considering uploading the **ord index** file `index.redb` (25gb currently) for others to just straight up download rather than syncing it. Obviously they'd need to index just the few blocks after my upload, but I imagine this would be a big help for others to get started on their inscribing journey. 

So...
- Is there any privacy concerns here (my own data attached)? 
- Is it only the `index.redb` file that would be needed?
- Would this work across operating systems? (I'm on a Mac).

Cheers.

**Comment by casey** [comment]
- There should be no privacy concerns, everyone's index.redb has the same content
- Only the index.redb file should be needed
- The underlying database, redb, is operating system agnostic, so it should work across platforms

However, this is unsupported, and users should only do this at their own risk. I'm worried that downloading an untrusted redb could be an attack vector. I've thought about whether we should provide an official way of downloading a recent `index.redb`, but I'm not sure if we should.

**Reply by DEFSTALKR** [reply]
Yeh good argument man, people would need to trust my link or even yours. In saying that, It would be a big help for onboarding though, and new people to be able to do it "at their own risk" from a third party (me) by bypassing the indexing which is a bit of a turn off right now IMO.
If people can download it from a semi-trusted source (the same 'known' link everytime) it might be a big step to get people onboarded, as taking one look in the discord, it's full of people who cannot index properly. 

I also agree that offering an 'official' link might be a bad idea as well. 

So as long as you don't have any issues with me offering it (i wont spam the discord haha), I'll see if I can get an upload up in the next day or so... But let me know if you would prefer me not to, not an issue either way.

**Reply by casey** [reply]
I think I don't want to do this as an official offering, but I'd love it if there was a community offering here. I don't want to encourage it, but if the options are people are going to do it by downloading random index.redbs from stranges, vs there being a good community option, the latter is definitely preferable.

**Reply by casey** [reply]
Just made a couple of related issues:

https://github.com/casey/ord/issues/1555
https://github.com/casey/ord/issues/1554

**Reply by casey** [reply]
A deterministic build signed by multiple community members would make recommending this much easier.
